index.js

```diff
@@ -1,17 +1,54 @@
-const fs = require('fs');
-const path = require('path');
+const request = require('request-promise');
```

Comment:
request and request-promise are deprecated, please use node-fetch. I know that A2 projects already complain about this deprecation but it will be easier to clean that up (which a client may require us to do soon) if we don't use it in new work.